typescript/infra/src/tx/govern-transaction-reader.ts

```diff
@@ -138,6 +149,11 @@ export class GovernTransactionReader {
       return this.readWarpModuleTransaction(chain, tx);
     }
 
+    // If it's an Ownable transaction
+    if (await this.isOwnableTransaction(chain, tx)) {
```

Comment:
this one's interesting because it's not quite mutually exclusive with the others, but we return early if we get a match for the others. Maybe just a sign that we should rethink the setup of this class at some point

Reply:
haha yeah. i actually added this catch-all for Ownables because we don't keep track of every ProxyAdmin in an artifact somewhere, so i noticed some warp tx parsing fail because it would not be able to determine what type of contract the ProxyAdmin was

maybe a sign to have better ProxyAdmin detection too